moto/cognitoidp/models.py

```diff
@@ -59,23 +59,25 @@ def to_json(self, extended=False):
 
         return user_pool_json
 
-    def create_jwt(self, client_id, username, expires_in=60 * 60, extra_data={}):
+    def create_jwt(self, username, expires_in=60 * 60, extra_data={}, audience=None):
         now = int(time.time())
         payload = {
             "iss": "https://cognito-idp.{}.amazonaws.com/{}".format(self.region, self.id),
             "sub": self.users[username].id,
-            "aud": client_id,
             "token_use": "id",
             "auth_time": now,
             "exp": now + expires_in,
             "email": self.users[username].username
         }
         payload.update(extra_data)
+        if audience:
+            payload['aud'] = audience
```

Comment:
For cognito generated tokens there is no aud for access tokens, it is there only for id token